listeners/tooltip.py

```diff
@@ -4,14 +4,25 @@
 
 class OmniSharpTooltipListener(sublime_plugin.EventListener):
 
-    def on_activated_async(self, view):
-        self._check_tooltip(view)
+    next_run_time = 0
 
-    def on_modified_async(self, view):
-        self._check_tooltip(view)
+    def on_activated(self, view):
+        self._check_tooltip_after_delay(view)
 
-    def on_selection_modified_async(self, view):
-        self._check_tooltip(view)
+    def on_modified(self, view):
+        self._check_tooltip_after_delay(view)
+
+    def on_selection_modified(self, view):
+        self._check_tooltip_after_delay(view)
+
+    def _check_tooltip_after_delay(self, view):
+        timeout_ms = 400
```

Comment:
They should be configurable, I haven't done that yet. It's 400 and 500 so that it's obvious they're different things and shouldn't be controlled by the same setting.